source/main_cr.f90

```diff
@@ -1161,7 +1135,7 @@ end subroutine envarsv
             oidpsv(ib2)=one/(one+dp1)
 ! Heavy ion variable
             moidpsv(ib2)=mtc(ib2)/(one+dp1)
-            nms(ib2)=m
+            nms(ib2)=1
```

Comment:
Shouldn't we get rid also of `nms(ib2)`? I haven't checked what this is for...

Reply:
I left it because it is used in postprocessing. I don't want to potentially start messing with the file formats.